docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -74,27 +71,27 @@ Inference with AUTO is configured similarly to other plugins: first you configur
 Following the OpenVINO™ naming convention, the Auto-Device plugin is assigned the label of “AUTO.” It may be defined with no additional parameters, resulting in defaults being used, or configured further with the following setup options: 
```

Comment:
the text above provide performance numbers, @myshevts is it valid to provide any performance in docs? I suppose some approvals are needed here

Secondly, can we move BA usage example to the end as for other plugins? In the intro we need to provide only information about what plugin does, no more

Reply:
I suggest to the the image (load-inference over timne) from the blog here

Reply:
this would obviate the need for the perf numbers